Resources/config/default_settings.yml

```diff
@@ -1,6 +1,8 @@
 parameters:
     ez_platformui.default.yui: {}
     ez_platformui.default.yui.filter: "min"
+    ez_platformui.default.yui.combine: true
+    ez_platformui.default.yui.build_dir: /bundles/ezplatformuiassets/vendors/yui3/build/
```

Comment:
Yeah... we've talked about it, and it's indeed better to favour consistency here. But the fact is that overall, I'm expecting bad surprises if somebody relies on the siteaccess awareness of PlatformUI :-)